dropwizard/service/build.gradle.kts

```diff
@@ -0,0 +1,187 @@
+
+  testImplementation("org.apache.iceberg:iceberg-api:${libs.versions.iceberg.get()}:tests")
+  testImplementation("org.apache.iceberg:iceberg-core:${libs.versions.iceberg.get()}:tests")
+  testImplementation("io.dropwizard:dropwizard-testing")
+  testImplementation(platform(libs.testcontainers.bom))
+  testImplementation("org.testcontainers:testcontainers")
+  testImplementation(libs.s3mock.testcontainers)
+
+  testImplementation("org.apache.iceberg:iceberg-spark-3.5_2.12")
+  testImplementation("org.apache.iceberg:iceberg-spark-extensions-3.5_2.12")
+  testImplementation("org.apache.spark:spark-sql_2.12:3.5.1") {
+    // exclude log4j dependencies
+    exclude("org.apache.logging.log4j", "log4j-slf4j2-impl")
+    exclude("org.apache.logging.log4j", "log4j-api")
+    exclude("org.apache.logging.log4j", "log4j-1.2-api")
+  }
+
+  testImplementation(platform(libs.awssdk.bom))
+  testImplementation("software.amazon.awssdk:glue")
+  testImplementation("software.amazon.awssdk:kms")
+  testImplementation("software.amazon.awssdk:dynamodb")
+
+  testImplementation(libs.auth0.jwt)
+
+  testCompileOnly(libs.smallrye.common.annotation)
+
+  testImplementation(platform(libs.junit.bom))
+  testImplementation("org.junit.jupiter:junit-jupiter")
+  testImplementation(libs.assertj.core)
+  testImplementation(libs.mockito.core)
+  testRuntimeOnly("org.junit.platform:junit-platform-launcher")
+
+  testImplementation(project(":polaris-eclipselink"))
+}
+
+if (project.properties.get("eclipseLink") == "true") {
+  dependencies { implementation(project(":polaris-eclipselink")) }
+}
+
+tasks.named<Test>("test").configure {
+  if (System.getenv("AWS_REGION") == null) {
+    environment("AWS_REGION", "us-west-2")
+  }
+  jvmArgs("--add-exports", "java.base/sun.nio.ch=ALL-UNNAMED")
+  useJUnitPlatform()
+  maxParallelForks = 4
+}
+
+tasks.register<JavaExec>("runApp").configure {
+  if (System.getenv("AWS_REGION") == null) {
+    environment("AWS_REGION", "us-west-2")
+  }
+  classpath = sourceSets["main"].runtimeClasspath
+  mainClass = "org.apache.polaris.service.dropwizard.PolarisApplication"
+  args("server", "$rootDir/polaris-server.yml")
+}
+
+application { mainClass = "org.apache.polaris.service.dropwizard.PolarisApplication" }
+
+tasks.named<Jar>("jar") {
+  manifest { attributes["Main-Class"] = "org.apache.polaris.service.dropwizard.PolarisApplication" }
+}
+
+tasks.register<Jar>("testJar") {
+  archiveClassifier.set("tests")
+  from(sourceSets.test.get().output)
+}
+
+val shadowJar =
+  tasks.named<ShadowJar>("shadowJar") {
+    manifest {
+      attributes["Main-Class"] = "org.apache.polaris.service.dropwizard.PolarisApplication"
+    }
+    mergeServiceFiles()
+    append("META-INF/hk2-locator/default")
+    isZip64 = true
+    finalizedBy("startScripts")
+  }
+
+val startScripts =
+  tasks.named<CreateStartScripts>("startScripts") {
+    classpath = files(provider { shadowJar.get().archiveFileName })
+    applicationName = "polaris-service"
```

Comment:
This is required to keep the script name to `polaris-service` and the environment variable to `$POLARIS_SERVICE_OPTS`

Reply:
Do we want to keep it as `polaris-service` or change it to `polaris-dropwizard-service` as well?
IMO the latter is likely better to not confuse people w/ Quarkus later.